### ISSUE TITLE
.MSG to PDF File Conversion

### ISSUE THREAD

**Issue by abin-aot** [bcgov/foi-flow#2977]
Title of ticket: 

#### Description
The purpose of this task is to convert .MSG Files arriving into Ministry Level S3 Bucket's request folder , to a "PROPERLY FORMATTED, ALIGNED" PDF file. We can refer the code base from File Watcher and create a Module for MSG/EML to PDF to plugin with a centralized .NET core microservices(#2975), which is intended to run on Linux based Docker container, in turn with our BC GOV OCP

Refer this repo for EXCEL conversion assembly : https://github.com/bcgov/foi-file-conversion

#### Dependencies
Are there any dependencies?

#### DOD
- [x]  Create a independent .NET core class library   for MSG (from S3) to PDF conversion (saved back to S3)- refer File Conversion Library
- [x]  Fetch the content of .MSG - in format of PDF - follow details of ICAL
- [x]  Create Unit Test for .MSG to PDF 
- [x]  Integrated with Main FOI DOC Conversion Microservice/computing service
- [x]  Tested with Docker
- [ ]

**Comment by KyEggleston** [comment]
Tested on ABC-3453-45345 in TEST-Marshal

- Was able to upload .MSG files to the Records tab.
- Records successfully ran through deduplication and file conversion (indicated by green checkmark for unique records, or a yellow duplicate icon for duplicates).
- Files did not change their file extension after file conversion (i.e., on the Records tab, their native file extension is displayed as a suffix on the name of the file).
- I was able to download the converted .MSG file as a PDF under the ellipsis (...) -> Download PDF. However, I was not able to open it. I received the following error when opening the file:

![image](https://user-images.githubusercontent.com/117312323/211896286-2eb98c65-065d-4032-8822-1c3452fa398f.png)

Related to ACs under https://github.com/bcgov/foi-flow/issues/3136.
